File: articles/algorithms-in-media-platforms-and-attention-systems/java/AttentionRisk.java
public class AttentionRisk {
    public static void main(String[] args) {
        double engagementPressure = 0.92;
        double creatorImpact = 0.88;
        double publicKnowledgeImpact = 0.78;
        double userControl = 0.44;
        double contestability = 0.42;
        double score = (engagementPressure + creatorImpact + publicKnowledgeImpact + (1.0 - userControl) + (1.0 - contestability)) / 5.0;
        System.out.printf("attention_risk_score=%.4f%n", score);
    }
}

File: articles/algorithms-in-media-platforms-and-attention-systems/fortran/attention_risk.f90
program attention_risk
  implicit none
  real :: engagement_pressure, creator_impact, public_knowledge_impact, user_control, contestability, score
  engagement_pressure = 0.92
  creator_impact = 0.88
  public_knowledge_impact = 0.78
  user_control = 0.44
  contestability = 0.42
  score = (engagement_pressure + creator_impact + public_knowledge_impact + (1.0 - user_control) + (1.0 - contestability)) / 5.0
  print *, "attention_risk_score=", score
end program attention_risk
